tenets/codelingo/go/ticker-in-for-select/codelingo.yaml

```
@@ -0,0 +1,33 @@
+tenets:
+  - name: ticker-in-for-select
+    actions:
+      codelingo/review:
+        comment: Make one ticker as opposed to making a new timer each iteration.
+    query: |
+      import codelingo/ast/go
+
+      any_of:
+        go.range_stmt(depth = any):
+          go.select_stmt(depth = any):
+            go.comm_clause(depth = any):
+              go.expr_stmt:
+                go.unary_expr:
+                  @review comment
+                  go.call_expr:
+                    go.selector_expr:
+                      go.ident:
+                        name == "time"
+                      go.ident:
+                        name == "After"
+        go.for_stmt(depth = any):
```

```suggestion
        // TODO: use path or macro facts for repeated code
        go.for_stmt(depth = any):
```

Path documentation is not merged yet [pr](https://github.com/codelingo/docs/pull/84), and macro facts aren't implemented.